src/Filter.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Filter from './Filter';
+import { shallow } from 'enzyme';
+
+  const mockColoradoClimbingAreas = {
+    "Canon City": {
+        "milesFromDenver": 119,
+        "id": 34,
+        "typesOfClimbing": ["trad", "sport", "toprope", "boulder"],
+        "classicRoutes": [
+            {
+                "name": "Enterprise",
+                "id": 35,
+                "grade": "5.09+",
+                "type": ["sport"],
+                "link": "https://www.mountainproject.com/route/105824687/enterprise"
+            },
+            {
+                "name": "Trick-or-Treat",
+                "id": 42,
+                "grade": "5.11a/b",
+                "type": ["toprope"],
+                "link": "https://www.mountainproject.com/route/105759012/trick-or-treat"
+            }
+        ]
+    },
+    "Boulder Canyon": {
+        "milesFromDenver": 31.5,
+            "id": 12,
+            "typesOfClimbing": ["trad", "sport", "toprope", "boulder", "aid"],
+            "classicRoutes": [
+                {
+                    "name": "North Face Center",
+                    "id": 13,
+                    "grade": "5.07+",
+                    "type": ["trad"],
+                    "link": "https://www.mountainproject.com/route/105748093/north-face-center"
+                },
+                {
+                    "name": "Empor",
+                    "id": 14,
+                    "grade": "5.07+",
+                    "type": ["trad"],
+                    "link": "https://www.mountainproject.com/route/105748142/empor"
+                }
+            ]
+        }
+    }
+
+
+  const mockFunction = jest.fn();
+
+
+
+
+
+
+describe('Filter', () => {
+
+  let wrapper;
+
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <Filter climb={mockColoradoClimbingAreas}
+                    updateFromFilter={mockFunction}/>
+      )
+  })
+
+  it('should test the states', () => {
+    expect(wrapper.state()).toEqual({setMax: 50, types: [], areas: [], areasPick: [], typesPick: []
+    })
+  })
+
+  it('should change type', () => {
+    expect(wrapper.state('setMax')).toEqual(50);
+    expect(wrapper.state('types')).toEqual([]);
+    wrapper.find('.range').simulate('change', {target: {value: 120}});
```

Comment:
Did you consider setting this value to be less than 119? It looks like you chose 120 so that your search would include both area objects but if you made the range grab just 1 of the area objects it would better demonstrate the filtering functionality since that is what this component's job.

Reply:
tried it with 115 and it worked. great idea! thanks